.bandit.yml

```diff
@@ -1,5 +1,5 @@
 ---
-skips: ["B101", "B107"]
+skips: ["B101", "B107", "B106"]
```

Comment:
B106: hardcoded_password_funcarg is an error that I do not think we should omit checking with bandit. Can't we add an exception for the directory / file if it is a test file? Please see lines 4-5 below. We do not want people to start... forgetting... hard coded passwords in the code base.

Reply:
I'll test again, but there was a test that was failing on the EOS.

Reply:
Removed from the exceptions.